manim/mobject/types/vectorized_mobject.py

```diff
@@ -846,11 +846,30 @@ def get_subcurve(self, a, b):
 
 class VGroup(VMobject):
     def __init__(self, *vmobjects, **kwargs):
-        if not all([isinstance(m, VMobject) for m in vmobjects]):
-            raise Exception("All submobjects must be of type VMobject")
         VMobject.__init__(self, **kwargs)
         self.add(*vmobjects)
 
+    def add(self, *vmobjects):
+        """Checks if all passed elements are an instance of VMobject and then add them to submobjects
+
+        Parameters
+        ----------
+        vmobjects :
```

Comment:
gotta tell us their type
```suggestion
        vmobjects : :class:`~.VMobject`
```

Reply:
By the way is there a particular way to treat star arguments in the docstring or is it fine this way ?

Reply:
@azarzadavila refer to the documentation guidelines for this, but: you document the type of the expected parameters.